File: scala/src/main/scala/io/jkratz/katas/gameoflife/Life.scala
package io.jkratz.katas.gameoflife

object Life {

  def main(args: Array[String]): Unit = {

    val state0 = Board.random(5,5)
    println("State 0")
    Board.prettyPrint(state0)

    val state1 = state0.evolve()
    println("State 1")
    Board.prettyPrint(state1)
  }
}

File: kotlin/src/main/kotlin/io/jkratz/katas/gameoflife/Life.kt
package io.jkratz.katas.gameoflife

object Life {

    @JvmStatic
    fun main(args: Array<String>) {
        val board = Board(10,10)
        println("State 0")
        board.prettyPrint()
        board.evolve()
        println("State 1")
        board.prettyPrint()
    }
}